dipy/viz/tests/test_fvtk.py

```diff
@@ -109,7 +118,18 @@ def test_colormap():
 @npt.dec.skipif(not fvtk.have_matplotlib)
 def test_colormaps_matplotlib():
     v = np.random.random(1000)
-    for name in 'jet', 'Blues', 'Accent', 'bone':
+    # The "Accent" colormap is deprecated as of 0.12:
+    with warnings.catch_warnings(record=True) as w:
+        accent_cm = data.get_cmap("Accent")
+        # Test that the deprecation warning was raised:
+        npt.assert_equal(len(w) > 0, True)
+
+    names = ['jet', 'Blues', 'bone']
+
+    if have_matplotlib and mpl_version < "2":
+        names.append('Accent')
+
+    for name in names:
```

Comment:
Well, the warning is raised regardless of if you are using MPL > 2 or not, because we are deprecating this cmap for all users. But now I also implemented a test that checks if the warning is raised in the loop below as well.